alpaca/common/rest.py

```diff
@@ -362,3 +364,72 @@ def _validate_credentials(
             )
 
         return api_key, secret_key, oauth_token
+
+    def get_marketdata(
```

Comment:
> Can we mark this as private?

I can, but I don't see much point of it.

> Also, how about adding api_version as parameter for the future possibility of having multiple api version in a single client? (i.e. possibility of adding v1beta endpoint?)

Let's add any additional parameters when they become needed.

Reply:
>> Can we mark this as private?
> I can, but I don't see much point of it.

Just because I do not want users to use get_marketdata directly instead of each functions currently.
also non-private method will be listed on the top of method list in vscode or IDE in my understanding.